tests/framework/extensions/clusters/clusters.go

```diff
@@ -915,3 +916,18 @@ func WatchAndWaitForCluster(steveClient *v1.Client, kubeProvisioningClient *kube
 	err = wait.WatchWait(result, IsProvisioningClusterReady)
 	return err
 }
+
+func GetProvisioningClusterByName(client *rancher.Client, clusterName string, namespace string) (*apisV1.Cluster, *steveV1.SteveAPIObject, error) {
```

Comment:
Sorry one last thing that I should have mentioned, but won't require another Jenkins run. For our generated docs we write comments for all our exported functions. In this format 
`// <Function> explanation of functions ...`
https://go.dev/doc/comment

Reply:
Done